mithril-common/src/chain_observer/cli_observer.rs

```diff
@@ -66,6 +66,7 @@ impl CardanoCliRunner {
     fn command_for_utxo(&self, address: &str, out_file: PathBuf) -> Command {
         let mut command = self.get_command();
         command
+            .arg("latest")
```

Comment:
It's not obvious that this is the cardano era that is used, plus since it's  used multiple times I think that you can add a `LATEST_CARDANO_ERA` static variable to encompass those concerns.

Reply:
You're right, I’ve addressed your comment in this [commit](https://github.com/input-output-hk/mithril/pull/2085/commits/8eea1e23309503e4bde6ab0ec0c574ab864f7dbe).